system/jlib/jstats.cpp

```diff
@@ -1869,6 +1869,21 @@ class CStatisticCollection : public CInterfaceOf<IStatisticCollection>
             cur.mergeInto(target);
     }
 
+    virtual void visit(IStatisticVisitor & visitor) const
+    {
+        if (visitor.visitScope(*this))
+        {
+            for (auto const & cur : children)
```

Comment:
Should this call visitChildren instead?

Reply:
I didn't because it would add an extra virtual call.  I thought about marking visitChilden as final, then it might get called directly.